src/pages/about.html

```diff
@@ -0,0 +1,174 @@
+        </li>
+        <li >
+            <a class="item" href="#Contact">
+            News
+          </a>
+        </li>
+        <li >
+          <a class="item item--button" href="#Contact">
+          Campaign
+        </a>
+      </li>
+      </ul>
+  </nav>    
+  <div class="header--content--about">
+    <div class="header--content--title--about">
+      <p class="header--content--title--greet">
+        "Hello! Sharing world"
+      </p>
+      <h1 class="header--content--title--main">“The best technology conference <br> on the planet”</h1>
+    </div>
+    <div class="header--content--main">
+    <p class="header--content-main--about">
+      Web Summit 2022 will bring together 70,000+ people, and the companies redefining the tech industry.
+    </p>
+    <p class="header-content-main-contact">
+      Please contact us per Email for any further questions <br>
+      about this website!<br>
+      <a href="#" class="contact--email">
+        habiburahmanpayenda@gmail.com
+      </a>
+    </p>
+    </div>
+  </div>
+  
+
+  </header>
+
+  <main class="main">
+    <section class="logo">
+      <h2 class="logo--title">Web<br> Global Summit 2022 Logo</h2>
+      <div class="borderLine"></div>
+      <p class="logo--text">
+        The logo of Web Summit 2022 was decidede through the Logo
+        compeition from 8 June to 7 July
+      </p>
+      <div class="logo--img"></div>
+    </section>
+
+    <section class="pastSummits">
+      <div class="pastSummits--first">
+      <h2 class="pastSummits--title">See the past Web Global Summits</h2>
+      <div class="borderLine"></div>
+      <p class="pastSummits--text">
+        Take a look at the last two Web Global Summits which took place in Tokyo and Madrid
+      </p>
+    </div>
+      <div class="pastSummits--seconde">
+      <div class="pastSummits--img1">
+        <h4 class="pastSummits--year">
+          2021
+        </h4>
+        <p class="pastSummits--location">
+          Web Global Summit 2013 in Tokyo
+        </p>
+      </div>
+      <div class="pastSummits--img2">
+        <h4 class="pastSummits--year">
+          2020
+        </h4>
+        <p class="pastSummits--location">
+          Web Global Summit 2013 in Madrid
+        </p>
+      </div>
+    </div>
+    </section>
+    <section class="partners">
+      <h3 class="partners--title">
+        Partners
+      </h3>
+      <div class="borderLine"></div>
+      <div class="partners--items">
+        <i class="fa-brands fa-firefox-browser item"></i>
+        <i class="fa-brands fa-google item "></i>
+        <i class="fa-brands fa-apple item "></i>
+        <i class="fa-brands fa-microsoft item "></i>
+        <i class="fa-brands fa-airbnb item "></i>
+      </div>
+    </section>
+  </main>
+  <footer class="footer">
+    <div class="footer--left"></div>
+    <div class="footer--right">
+      <p class="footer--right--text">
+        2022 Web Summit 
+        Some Rights Reserved
+      </p>
+    </div>
+  </footer>
```

Comment:
- [ ] Based on the requirements of this project, the footer section of the about page for the desktop version should have a dark background. Kindly fix this too since the previous reviewer forgot this requirement.